From: Ricardo Neri <ricardo.neri@ti.com>
Date: Thu, 24 Feb 2011 16:08:48 +0000 (-0600)
Subject: OMAP4: HDMI: Add borders in porches of VESA4 timings
X-Git-Tag: android-2.6.35-2.3-omap4.13.1~40
X-Git-Url: http://git.omapzoom.org/?p=kernel%2Fomap.git;a=commitdiff_plain;h=9ffcabc5f5bc3c350bf9eab14eb86599fb58c215

OMAP4: HDMI: Add borders in porches of VESA4 timings

VESA4 specifies borders of 8 pixels at the right and left and
8 lines at the top and bottom of the Addressable Video region. Such
borders are included in the vertical/horizontal front/back porches.

See VESA Monitor Timing Standard for further details.

Change-Id: Id79c358e022d79f1fc81de863474d5f7024e4aa9
Signed-off-by: Ricardo Neri <ricardo.neri@ti.com>
---

diff --git a/drivers/video/omap2/dss/hdmi.c b/drivers/video/omap2/dss/hdmi.c
index c156853..9d28704 100644
--- a/drivers/video/omap2/dss/hdmi.c
+++ b/drivers/video/omap2/dss/hdmi.c
@@ -198,7 +198,7 @@ static const struct omap_video_timings all_timings_direct[] = {
 	{2880, 480, 108108, 248, 64, 240, 6, 9, 30},
 	{1920, 1080, 74250, 44, 638, 148, 5, 4, 36},
 	/* Vesa frome here */
-	{640, 480, 25175, 96, 8, 40, 2, 2, 25},
+	{640, 480, 25175, 96, 16, 48, 2, 10, 33},
 	{800, 600, 40000, 128, 40, 88, 4 , 1, 23},
 	{848, 480, 33750, 112, 16, 112, 8 , 6, 23},
 	{1280, 768, 79500, 128, 64, 192, 7 , 3, 20},